contribs/gmf/less/layertree.less

```diff
@@ -10,9 +12,19 @@ gmf-layertree ul .treenode {
     padding-left: 0;
     text-decoration: none;
     &::after {
-      display:none;
+      display: none;
+    }
+
+    &.expand-node.fa {
+      &:before {
```

Comment:
`::before`
Though it works with a single ':', it's better to use the CSS3 notation with a double.